src/resources/views/index/job-form.blade.php

```diff
@@ -190,6 +190,22 @@
                 </span>
             @endif
         </div>
+
+        <div class="form-group{{ $errors->has('oa_ignore_existing_label') ? ' has-error' : '' }}">
+            <div class="checkbox">
+                <label for="oa_ignore_existing_label">
+                    <input id="oa_ignore_existing_label" type="checkbox" name="oa_ignore_existing_label" value="1" @checked(old('oa_ignore_existing_label'))>
+                    Ignore Existing Labels
+                </label>
+            </div>
+            @if ($errors->has('oa_ignore_existing_label'))
+               <span class="help-block">{{ $errors->first('oa_ignore_existing_label') }}</span>
+            @else
+                <span class="help-block">
+                    Ignore existing annotations label as labels for training proposals. By default, Existing annotations labels are used
```

Comment:
Please be careful to use correct spelling. This could also be clarified to:
```suggestion
                    Ignore labels of existing annotations. Training proposals will have no labels if this is enabled.
```